test/bigint_test.dart

```diff
@@ -0,0 +1,36 @@
+//
+// bigint_test.dart
+// Test of very wide bitvector comparison failure
+//
+//
+
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
I think a test that could easily reproduce is just performing operations on a `LogicValue` directly without any actual hardware.  Worth adding that one?  I think any >64-bit `LogicValue` with any operation (e.g. `+`) against another `LogicValue` which is "filled" (all the same value, like all 1's) should be sufficient.